mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -31,21 +77,64 @@ AUI.add(
 									var listItem = currentTarget.ancestor('li');
 
 									listItem.remove();
+
+									if (finishedTask) {
+
+										var finishedHtml = A.Lang.sub(
+											TPL_FINISHED_TASK,
+											{
+												taskFinished: instance._taskFinished
+											}
```

Comment:
You are referring to config value that does not exist yet. It is in second PR, https://github.com/markocikos/liferay-internship/pull/151/commits/e1d0abfa776b9362be29c8019ebdc5a8c1e99742, but localization implementation is wrong there.

1. Have a complete feature withing one PR.
1. Implement localization correctly, meaning definition in ```Lanaguage.properties``` file and usage of ```UnicodeLanguageUtil()``` in JSP.